src/rpcmasternode.cpp

```diff
@@ -265,6 +265,19 @@ UniValue masternode(const UniValue& params, bool fHelp)
 
                 bool fResult = CMasternodeBroadcast::Create(mne.getIp(), mne.getPrivKey(), mne.getTxHash(), mne.getOutputIndex(), strError, mnb);
 
+                CPubKey pubKeyMasternode;
+                CKey keyMasternode;
+                if(darkSendSigner.GetKeysFromSecret(mne.getPrivKey(), keyMasternode, pubKeyMasternode)) {
+                    LogPrint("masternode", "masternode start-alias -- alias = %s, pubKeyMasternode.GetID() = %s\n",
```

Comment:
Log output like this should rather be a part of `CMasternodeBroadcast::Create` imo because:
1) `GetKeysFromSecret` is already called there, no need to duplicate
2) this logic itself is/should not be `start-alias` only related, same logic applies to start/start-*